service/src/main/java/bio/terra/profile/service/profile/flight/create/CreateProfileFlight.java

```diff
@@ -39,7 +39,9 @@ public CreateProfileFlight(FlightMap inputParameters, Object applicationContext)
     addStep(new CreateProfileStep(profileDao, billingProfile, user));
     switch (billingProfile.cloudPlatform()) {
       case GCP:
-        addStep(new CreateProfileVerifyAccountStep(crlService, billingProfile, user));
+        addStep(
```

Comment:
Not sure I understand. Why are we changing create profile logic? How this is related to update profile?

Reply:
I renamed the step when there was a separate `UpdateProfileFlight`. Now that there isn't a separate `UpdateProfileFlight`, I'll revert the name change.

The step itself was changed to use the new `GcpService` so we can reuse code in the step and in the `ProfileService`.